src/components/Tree/TreeItem/DragHandle.tsx

```diff
@@ -15,6 +15,7 @@ export const DragHandle = forwardRef(
     ({ active, className, ...props }: DragHandleProps, ref: React.ForwardedRef<HTMLButtonElement>) => {
         return (
             <button
+                aria-labelledby="DraggableLabel"
```

Comment:
We don't set this ID anywhere in Fondue, why not using static `aria-label`? (translations is a topic for the Fondue team and out-of-scope)

Reply:
exactly because of that, there is no translations yet in fondue. The label can be controlled from clarify depending on the context, language and needs

Reply:
We should rely on:
```
onDragStart({ active }) {
    return `Picked up ${getActiveTitle(active) || active.id}.`;
},
```

Translations for later as none of the components have translations yet.

Reply:
one thing is the aria-description and a different is the aria-label. Accessibility tests complains about the missing label... I can add a static label `Draggable Item` and leave with that for now

Reply:
done

Reply:
closing clarify PR https://github.com/Frontify/clarify/pull/15090 since it's not needed anymore